index.bs

```diff
@@ -14285,7 +14343,7 @@ A {{DOMException}} is represented by a
 
 <div algorithm="to create a simple exception or DOMException">
 
-    To [=create=] a [=simple exception=] or {{DOMException}} |E|, with a string giving the
+    To create a [=simple exception=] or {{DOMException}} |E|, with a string giving the
```

Comment:
The "create" term is now ambiguous and this didn't feel like it was referencing the existing "create" to me so I deleted it instead of disambiguating.